dataRetriever/factory/resolverscontainer/baseResolversContainerFactory.go

```diff
@@ -309,34 +316,17 @@ func (brcf *baseResolversContainerFactory) generatePeerAuthenticationResolver()
 	return brcf.container.Add(identifierPeerAuth, peerAuthResolver)
 }
 
-func (brcf *baseResolversContainerFactory) createOneResolverSender(
-	topic string,
-	excludedTopic string,
-	targetShardId uint32,
-) (dataRetriever.TopicResolverSender, error) {
-	return brcf.createOneResolverSenderWithSpecifiedNumRequests(
-		topic,
-		excludedTopic,
-		targetShardId,
-		brcf.numCrossShardPeers,
-		brcf.numIntraShardPeers,
-		brcf.numFullHistoryPeers,
-		brcf.currentNetworkEpochProvider,
-	)
-}
-
 func (brcf *baseResolversContainerFactory) createOneResolverSenderWithSpecifiedNumRequests(
 	topic string,
 	excludedTopic string,
 	targetShardId uint32,
-	numCrossShard int,
-	numIntraShard int,
-	numFullHistory int,
-	currentNetworkEpochProvider dataRetriever.CurrentNetworkEpochProviderHandler,
+	numCrossShardPeers int,
```

Comment:
why do we have to provide the numCrossShardPeers value?
Please check if it is always passed as brcf.numCrossShardPeers and if yes, might remove the parameter

Reply:
`generateTrieNodesResolvers` methods call it with 0 `numCrossShardPeers` and all `numTotalPeers` intra